cvat/apps/engine/static/engine/js/shapes.js

```diff
@@ -3772,6 +3776,9 @@ class RaysView extends PolyShapeView {
     }
 
     _isVanishingPointVisible() {
+        if (!this._controller._model._rotateRays)
```

Comment:
why does the option to rotate lines affects the visibility of the vanishing point?
Vanishing point is defined as the intersection of left and right lines

Reply:
Initially vanishing point used to rotate left/center/right by keeping center line as bisector. Since this functionality is disabled, vanishing point lost its value and therefore no benefit to visualise it.